docs/configure.rst

```diff
@@ -139,12 +139,28 @@ The above code should be styled according to the
 Mark a code cell as read-only
 =============================
 
-If you'd like a code cell to be runnable by Thebe, but not *editable* by the user, you
+If you would like a code cell to be runnable by Thebe, but not *editable* by the user, you
 may mark it as "read-only" with the following syntax:
 
 .. code-block:: html
 
    <pre data-executable data-readonly>print("I cannot be modified")</pre>
 
-Users will not be able to modify the code when Thebe is activated, though they can still
+Users will not be able to modify the code once Thebe is activated, though they can still
 press the "run" button to see the outputs.
+
+**To set all cells as read-only by default**, use the following `thebe` configuration:
+
+.. code:: javascript
+
+   codeMirrorConfig: {
+       readOnly: true
+   }
+
+This uses codeMirror to mark all cells as read-only. If you are using this setting and would like to
+manually mark individual cells as editable, you can override the codeMirror configuration for a cell using `data-readonly="false"`. For example:
```

Comment:
I can't figure out why \`data-readonly="false"\` isn't rendering as `data-readonly="false"` here for inline code. Any ideas? I think this is the last outstanding issue for this PR.

Reply:
ah my bad - it was because I used a single backtick and rST requires two backticks by default